src/main/java/com/elixr/poc/service/PurchaseService.java

```diff
@@ -0,0 +1,29 @@
+package com.elixr.poc.service;
+
+import com.elixr.poc.Repository.PurchaseRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+@Service
+public class PurchaseService {
+    private final PurchaseRepository purchaseRepository;
+
+    public PurchaseService(PurchaseRepository purchaseRepository) {
+        this.purchaseRepository = purchaseRepository;
+    }
+
+    /*
+    Deleting the purchase from the database
+     */
+    public boolean deletePurchaseDetails(UUID purchaseId) {
+        boolean success = false;
+        try {
```

Comment:
Its better to check if entity exists by calling repository.findById(uuid) first.
If not found then display an error message "No matching entity found"